lib/core/src/lib/datatable/components/datatable/datatable.component.scss

```diff
@@ -350,7 +350,14 @@ $data-table-cell-min-width-file-size: $data-table-cell-min-width-1 !default;
     }
 
     .adf-datatable-row:not(:hover) .adf-datatable-hide-actions-without-hover {
-        display: none;
+        visibility: hidden;
+    }
+
+    .adf-datatable-actions-menu:focus-visible,
```

Comment:
I see in 323rd line there is already rule for .adf-datatable-actions-menu so maybe we can use '&' operator to have shorter selector?

Reply:
Good point, changed